client/stylesheets/_toCat.scss

```diff
@@ -4330,4 +4330,153 @@ On click the images will be displayed at normal size to complete the effect
 
 
 
+/* Notif */
+
+.notifMainWrap {
+    width: 70%;
+    text-align: left;
+}
+
+.notifLi {
+    width: 100%;
+}
+
+.notifLi ul > li {
+    display: inline-block;
+}
+
+.notifLi {
+    margin: 2px 0;
+    padding: 10px 0;
+    border-bottom: 1px solid #d3d3d7;
+}
+
+.notifImage img, .notifInfo {
```

Comment:
Selector `notifImage` should be written in lowercase with hyphens<br>Selector `notifInfo` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line